### PR TITLE
fix(platform-browser): setAttribute should work with xmlns namespace

### LINKED ISSUE #14865: Failed to execute 'setAttributeNS' on 'Element': '' - on xmlns:v="..."
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
```

**Current behavior**
given this template (a breadcrumb with rdf attributes for search engines):
```twig
<ol class="breadcrumb" xmlns:v="http://rdf.data-vocabulary.org/#">
    <li typeof="v:Breadcrumb">
        <a href="/foo" rel="v:url" property="v:title">foo</a>
    </li>
</ol>
```

angular 2.4.9 - works: https://plnkr.co/edit/1Wyd4n1t8RPw7JTC4nYe?p=preview
angular 4.0.0-rc.2 - throws: https://plnkr.co/edit/qBc4ODqbBkPlMDrQVNf8?p=preview

Error:
image

### PR DESCRIPTION
**Pull request by DzmitryShylovich**
Closes #14865

### CODE REVIEW

File: modules/@angular/platform-browser/src/dom/dom_renderer.ts [hunk 1]
```diff
@@ -149,7 +149,13 @@ class DefaultDomRendererV2 implements RendererV2 {
 
   setAttribute(el: any, name: string, value: string, namespace?: string): void {
     if (namespace) {
-      el.setAttributeNS(NAMESPACE_URIS[namespace], namespace + ':' + name, value);
+      const attrNs = NAMESPACE_URIS[namespace];
```
Comment (vicb) [inline modules/@angular/platform-browser/src/dom/dom_renderer.ts]: what about adding the xmlns NS ? (This should be the [list](https://www.w3.org/TR/html5/infrastructure.html#namespaces) of all supported NS).
missing tests

**Comment by DzmitryShylovich** [comment]
@vicb added uri for `xmlns` namespace + tests PTAL

**Comment by DzmitryShylovich** [comment]
@IgorMinar can you take a look please? it's a regression from 2.x

**Comment by patrickmichalina** [comment]
Can this be merged in before the next RC? It is preventing any migration to 4.0.0.RC for my projects.

**Comment by SamVerschueren** [comment]
Same here, have to downgrade back to `2.x` after noticing this in one of my views.

**Review by tbosch — APPROVED** [review]
Thanks, this is a good change!

**Comment by SamVerschueren** [comment]
Thanks @DzmitryShylovich for fixing it and thanks @IgorMinar and @vicb for following up! 🍻

**Pull request merged**